src/plugins/lua/bayes_expiry.lua

```diff
@@ -207,7 +207,7 @@ local function expire_step(cls, ev_base, worker)
   lredis.exec_redis_script(cls.script,
       {ev_base = ev_base, is_write = true},
       redis_step_cb,
-      {'RS*_*', cls.expiry, cur}
+      {3, 'RS*_*', cls.expiry, cur}
```

Comment:
I don't understand why do you add this complexity. `fann_redis` will also be broken after this commit.

Reply:
`fann_redis` does not use `exec_redis_script`; I don't see how it could be broken.

Reply:
It does. I still don't understand why have you changed it by the way.

Reply:
the reason for adding this parameter is to deal with the change in `lua_redis.lua`

Reply:
I see, I managed to miss it somehow. I can fix this. I could also use key arguments only everywhere.